geniza/corpus/models.py

```diff
@@ -1126,6 +1136,16 @@ def index_data(self):
                 content = fn.content_html_str
                 if content:
                     transcription_texts.append(Footnote.explicit_line_numbers(content))
+            elif Footnote.DIGITAL_TRANSLATION in fn.doc_relation:
+                content = fn.content_html_str
+                if content:
+                    translation_texts.append(Footnote.explicit_line_numbers(content))
+                    # TODO: How to handle multiple translations in different languages?
```

Comment:
Since translations are into modern languages (I think? wondering about Goitein's translations into Hebrew), you may want to index them separately by language. Not sure how many languages we're including or how complicated it will get. You probably don't need to do it on the first pass! (Maybe we don't do it for all languages but a subset where we have enough that it matters / prioritized by the team).

I'm imagining something like `translation_text_en`, `translation_text_de`, `translation_text_he` using dynamic field types, and then should be possible to search across them using wildcard field names (I think, haven't done it myself in practice).

Reply:
Interesting idea! The reason I made the comment was that I was wondering how to handle indexing `translation_language_code_s` when there are multiple translations so that they can get the `lang` and `dir` attributes appropriately in HTML. That would definitely solve that problem—are there other advantages of indexing them separately by language? Leveraging stemmers and the like for each?

Reply:
yes, exactly - will allow language-specific searching and indexing (hence my comment about modern languages: I think there will be reasonable Solr support for most of the translation languages)